### DISCUSSION TITLE
挂载阿里云盘open时一直报错

### DISCUSSION THREAD

**Discussion by ENDDREAM-ZM** [AlistGo/alist#6738] [Q&A]
导入恢复和手动添加挂载一直显示Failed init storage: failed to refresh token: Too Many Requests
然而即使等了一个小时以后再保存还是这样

**Comment by ENDDREAM-ZM** [comment]
在我刚发出来一分钟后突然就好了 在此之前等多久都这样 好奇怪

**Reply by AspirantH** [reply]
今天我也遇到了，不知道怎么解决
